platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/metrics/SyncMetrics.java

```diff
@@ -65,6 +70,37 @@ public class SyncMetrics {
                     INTERNAL_CATEGORY, PlatformStatNames.TIPS_PER_SYNC)
             .withDescription("the average number of tips per sync at the start of each sync")
             .withFormat(FORMAT_15_3);
+
+    private static final SpeedometerMetric.Config INCOMING_SYNC_REQUESTS_CONFIG = new SpeedometerMetric.Config(
+                    PLATFORM_CATEGORY, "incomingSyncRequests/sec")
+            .withDescription("Incoming sync requests received per second")
+            .withFormat(FORMAT_14_7);
+    private final SpeedometerMetric incomingSyncRequestsPerSec;
+
+    private static final SpeedometerMetric.Config ACCEPTED_SYNC_REQUESTS_CONFIG = new SpeedometerMetric.Config(
+                    PLATFORM_CATEGORY, "acceptedSyncRequests/sec")
+            .withDescription("Incoming sync requests accepted per second")
+            .withFormat(FORMAT_14_7);
+    private final SpeedometerMetric acceptedSyncRequestsPerSec;
+
+    private static final SpeedometerMetric.Config OPPORTUNITIES_TO_INITIATE_SYNC_CONFIG = new SpeedometerMetric.Config(
+                    PLATFORM_CATEGORY, "opportunitiesToInitiateSync/sec")
+            .withDescription("Opportunities to initiate an outgoing sync per second")
+            .withFormat(FORMAT_14_7);
+    private final SpeedometerMetric opportunitiesToInitiateSyncPerSec;
+
+    private static final SpeedometerMetric.Config OUTGOING_SYNC_REQUESTS_CONFIG = new SpeedometerMetric.Config(
+                    PLATFORM_CATEGORY, "outgoingSyncRequests/sec")
+            .withDescription("Outgoing sync requests sent per second")
+            .withFormat(FORMAT_14_7);
+    private final SpeedometerMetric outgoingSyncRequestsPerSec;
+
+    private static final SpeedometerMetric.Config SYNCS_PER_SECOND_CONFIG = new SpeedometerMetric.Config(
+                    PLATFORM_CATEGORY, "syncs/sec")
+            .withDescription("Total number of syncs completed per second")
+            .withFormat(FORMAT_14_7);
+    private final SpeedometerMetric syncsPerSec;
```

Comment:
I would use a `CountPerSecond` metric for this, its more accurate and lightweight

Reply:
`CountPerSecond` does not extend `Metric` yet, which means it cannot be exported to the CSV, right?

Reply:
Done